smrf/data/load_topo.py

```diff
@@ -104,18 +104,11 @@ def readImages(self, f):
         # to double or int
         for v_smrf in self.IMAGES:
 
-            # check to see if the user defined any variables
-            # e.g. veg_height = veg_length
```

Comment:
This has never been used where one can specify different field names in the topo.nc. Most will be generated with `basin_setup` so it's just making it a little more rigid.